src/lib/_imports/trumps/s-hero-banner.css

```diff
@@ -0,0 +1,188 @@
+    position: absolute;
+    height: auto;
+    width: 475px;
+    left: 0;
+    bottom: 50px;
+  }
+}
+.o-section--banner .s-home__banner .o-section__banner-overlay {
+  @extend %x-overlay--callout;
+
+  --color-text: inherit;
+  --color-bkgd-rgb: var(--global-color-primary--normal);
+
+  color: var(--color-text);
+  background-color: rgba(var(--color-bkgd-rgb), 0.9);
+}
+.o-section--banner.o-section--style-dark .s-home__banner .o-section__banner-overlay {
+  --color-text: var(--global-color-primary--xx-light);
+  --color-bkgd-rgb: var(--global-color-primary--xx-dark-rgb);
+}
+.o-section--banner.o-section--style-light .s-home__banner .o-section__banner-overlay {
+  --color-text: var(--global-color-primary--xx-dark);
+  --color-bkgd-rgb: var(--global-color-primary--xx-light-rgb);
+}
+
+
+
+/* Banner: Image */
+
+/* Temporary fix to get rid of white border on top of banner */
+.o-section--banner .o-section__banner-image {
+  top: 0%;
+  transform: translate(-50%, 0%);
+}
+
+
+
+
+
+/* Introduction */
+
+.s-home__intro {
+  --left-buffer: 48px;
+
+  padding: 24px 84px 36px var(--left-buffer);
+
+  font-weight: bold;
+}
+
+.s-home__intro h1 {
+  margin-top: 0;
+  padding-bottom: 14px;
+  margin-bottom: 18px;
+
+  font-size: 2.4rem; /* design deviates from standard to fit text on 2 lines */
+
+  border-bottom: var(--global-border-width--normal) solid var(--global-color-primary--xx-dark);
+}
+
+
+
+
+
+/* Content */
+
+
+
+/* Content: Panels */
+
+/* First panel */
+@media (--wide-and-above) {
+  /* To match design */
+  .s-home__content > *:nth-child(1) { margin-right: 100px; }
+}
+@media (--medium-and-above) and (--wide-and-below) {
+  /* To prevent wrapping of `.s-focus-list` items */
+  .s-home__content > *:nth-child(1) { margin-right: 25px; }
+}
+
+/* Second panel */
+@media (--medium-and-above) {
+  .s-home__content > *:nth-child(2) {
+    /* To draw a border between grid items (that has existing gap) */
+    border-left:
+      var(--global-border-width--normal) solid
+      var(--global-color-primary--xx-dark);
+    --border-color: var(--global-color-primary--xx-dark);
+    padding-left: var(--gap);
+
+    /* To allow text to spill into right-hand margin of content */
+    margin-right: -35px; /* anywhere between when text wraps: -10px and -43px */
+  }
+}
+
+
+
+/* Content: Elements */
+
+.s-home__content h2 {
+  font-size: var(--global-font-size--large);
+}
+.s-home__content p {
+  font-weight: var(--medium);
+}
+
+
+
+/* Content: Focus List */
+
+.s-home__content .s-focus-list {
+  font-size: var(--global-font-size--large);
+}
+
+
+
+/* Content: Icon List */
+
+.s-home__content .s-icon-list {
+  padding-left: 0;
+  margin-block: 46px;
+
+  /* line-height: 1.75; */
+
+  color: var(--global-color-primary--xx-dark);
+}
+.s-home__content .s-icon-list li + li {
+  margin-top: 40px;
+}
+@media (--wide-and-below) and (--medium-and-above) {
+  .s-home__content .s-icon-list {
+    --icon-size: 2em;
+  }
+}
```

Comment:
> [!CAUTION]
> **Unused code.** Please delete.

The `.s-icon-list` and `.s-focus-list` classes are irrelevant to this pattern. Notice how you do not use them in the HTML.